src/ngraph/op/util/op_annotations.hpp

```diff
@@ -25,6 +25,19 @@ namespace ngraph
             /// \brief Abstract base class for annotations added to graph ops
             class OpAnnotations
             {
+            public:
+                void set_in_place_oi_pairs(const std::map<size_t, size_t>& oi_pairs)
```

Comment:
Not sure what an oi_pair is. What is it? Can we name this better? If not, can we add a comment?